package.json

```diff
@@ -40,5 +51,34 @@
   },
   "dependencies": {
     "mobx": "^5.9.0"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "release": {
+    "branch": "develop",
+    "verifyConditions": [
+      "@semantic-release/git",
+      "@semantic-release/npm"
+    ],
+    "prepare": [
+      "@semantic-release/git",
```

Comment:
Would it be possible to move all this stuff in `.releaserc`? That'd help us to keep `package.json` clean